docs/troubleshooting-guide/saml.md

```diff
@@ -8,6 +8,10 @@ title: Troubleshooting SAML authentication setup
 
 Some common problems when setting up SAML.
 
+## Does your app support SAML?
+
+Verify that the application you created in your IdP supports SAML. Sometimes other options are presented during the app creation process.
```

Comment:
Like what other options? Can we give examples?

Reply:
I moved this paragraph from "Troubleshooting tips" (originally in `authenticating-with-saml.md`), so  I don't have this info on hand.

[Added a note to the backlog](https://www.notion.so/metabase/Updates-by-topic-parent-ticket-5f93c492e1b8483ab5500768b10af0f9#9d4cf3ede11a40cdae702eeeb50a6592) for now :)